docs/src/main/paradox/java/http/routing-dsl/directives/parameter-directives/parameters.md

```diff
@@ -3,10 +3,13 @@
 
 Extracts multiple *query* parameter values from the request.
 
+If an unmarshaller throws an exception while extracting the value of a parameter, it will be handled as a rejection.
```

Comment:
Can you also change it to the more accurate text I posted for the Scala case below?